tools/superdsm/superdsm.xml

```diff
@@ -72,7 +89,27 @@
     </environment_variables>
     <inputs>
         <param name="dataset" type="data" format="tiff,png" label="Dataset" />
-        <param name="seg_border" type="integer" min="1" value="8" label="Width of the outlines (in pixels) of the segmentation results (overlays)" />
+        <section name="outputs" title="Outputs" expanded="true">
+            <conditional name="do_overlay">
+                <param name="do_overlay_enabled" type="select" label="Segmentation overlay">
+                    <option value="true" selected="true">Create a segmentation overlay</option>
+                    <option value="false">Do not create a segmentation overlay</option>
+                </param>
+                <when value="true">
+                    <param name="seg_border" type="integer" min="1" value="8" label="Width of the outlines (in pixels)" />
+                </when>
+                <when value="false">
+                </when>
+            </conditional>
+            <param name="do_masks_enabled" type="select" label="Label map">

```

Comment:
You could model that differently. Can you not put all 3 in one conditional. 1) Create a segmentation overlay 2) Create a label map 3) Report all hyperparameters

Reply:
There are 3 optional outputs, which gives us 7 valid combinations (a, b, c, ab, ac, bc, abc). I think that, from an UX perspective, ticking each of the outputs individually is preferable (the user only has to read through 3 options, not through 7). Sure, submitting a job which produces no outputs makes no sense and a good UX pattern shouldn't allow this — so neither of the two solutions is optimal. I can change that as you suggest, but my personal opinion is that, from an UX perspective, having only 3 instead of 7 options outweighs preventing an invalid option (which still produces the output requested by the user, namely an empty one).

Are there any UX experts around whom we might ask?